design/release-notes/release-0.11/draft-release-notes.md

```diff
@@ -85,12 +86,57 @@ users secure services running on Kubernetes.
 
 ### New CertificateRequest resource type
 
+With this release comes the graduation of the `CertificateRequest` resource from
+alpha to beta. This means that when requesting certificates using the
```

Comment:
We've not actually graduated the CertificateRequest resource type yet to beta, but I see why you've said this.

This raises the point that we should probably be more consistent with our alpha/beta/ga terminology.

Can we rephrase this simply to say it's been enabled by default?